client/src/test/java/com/flipkart/flux/client/FluxClientSpyModuleForTests.java

```diff
@@ -35,21 +35,26 @@
     @Override
     protected void configure() {
         install(new FluxClientInterceptorModule());
-        bind(ExecutableRegistry.class).annotatedWith(IsolatedEnv.class).to(LocalExecutableRegistryImpl.class);
         bind(SimpleWorkflowForTest.class);
         install(new TestResourceModule());
     }
 
     @Provides
     @Singleton
-    public LocalContext provideLocalContext( ) {
+    public LocalContext provideLocalContext() {
         return Mockito.spy(new LocalContext());
     }
 
+
+    public AuthTokenService testAuthTokenService(){
+        return  Mockito.spy(new  AuthTokenService("https://authn-preprod.nm.flipkart.com",
+                new OAuthClientCredentials("http://localhost:9998/", "admJWTjYAefdTBvcFgPOmE4Km1zDLzm7wESzkpOPkwhXlCXm")));
```

Comment:
If possible, can we read this from config file ? It will avoid duplication of clientSecret at multiple places.